packages/amplication-client/src/routes/appRoutes.tsx

```diff
@@ -75,7 +76,7 @@ export const Routes: RouteDef[] = [
     exactPath: true,
   },
   {
-    path: "/github-auth-app/callback",
+    path: "/github-auth-app/callback", //todo
```

Comment:
Remove the "todo" :)

Reply:
@abrl91  Because this branch is not my final work on routes i prefer to leave this comment to my new branch :)